client/core/core.go

```diff
@@ -7743,10 +7743,9 @@ func (c *Core) fetchFiatExchangeRates() {
 	c.wg.Add(1)
 	go func() {
 		defer c.wg.Done()
-		tick := time.NewTimer(fiatRateRequestInterval)
+		tick := time.NewTicker(fiatRateRequestInterval)
 		defer tick.Stop()
 		for {
-
 			c.refreshFiatRates(ctx)
 
```

Comment:
Generally, I think it's better to create a new timer on every loop, so that it is X time **between the end** of one call and the start of the other. This prevents any chance that calls can pile up. In this case, since `fiatRequestInterval` is much greater than a 4 second timeout internally in `refreshFiatRates`, I don't think we'll have any problems with this. 

I actually thought that @ukane-philemon was creating a new timer every loop here. Wasn't there a method calculated the next tick based on the current state somehow? Maybe I'm confused.

Anyway, this is fine, I guess, Just wanted to comment on the different patterns.

Reply:
Yeah, this was just how I thought it should be (consistent time between requests, not time between end of one call and start of another).  But I'd be fine with a new timer on each iteration if you guys really think it makes a difference.  This is just simple and regular request intervals.

Reply:
For ref: https://github.com/decred/dcrdex/pull/1600#discussion_r921498759
`nextRateSourceTick` added nothing but complexity so we removed it.  It's still straightforward to make a new *timer* in the loop though.  No need for a helper function because all the last request recording for the different sources was simplified too.

Reply:
As long as that 4 second timeout is effective, it's fine as is.

Reply:
That's not how a ticker works though.  It won't spew out multiple signals if no one was listening for a while.  The internal channel sent on a ticker (or timer) is non-blocking.

Reply:
See https://pkg.go.dev/time#NewTicker

> The ticker will adjust the time interval or drop ticks to make up for slow receivers

It doesn't matter if the timeout internal to `refreshFiatRates` is effective.